### PR TITLE
Ported some includes over to new syntax

### PR DESCRIPTION
**Pull request by WildCard65**
The physconic commit was put in this from accidently putting 1 too many for HEAD~

### CODE REVIEW

File: plugins/include/entity.inc [hunk 1]
```diff
@@ -615,10 +613,10 @@ native SetEntPropEnt(entity, PropType:type, const String:prop[], other, element=
  * @error				Invalid entity, property not found, or property not 
  *						actually a vector data type.
  */
-native GetEntPropVector(entity, PropType:type, const String:prop[], Float:vec[3], element=0);
+native void GetEntPropVector(int entity, PropType type, const char[] prop, float vec[3], int element=0);
 
 /**
- * Sets a vector of floats in an entity, given a named network property.
+ * Sets a vector of float s in an entity, given a named network property.
```
Comment (yedpodtrzitko) [inline plugins/include/entity.inc]: this looks like an accidental change

**Comment by yedpodtrzitko** [comment]
looks good (except the small typo)

File: plugins/include/entity.inc (hunk 1 above)
Comment (WildCard65) [inline plugins/include/entity.inc]: Ya it was.

**Comment by powerlord** [comment]
I should probably point out that I already ported all the map-related plugins and headers in pull request #226

**Comment by WildCard65** [comment]
I kind of forgot about your PR @powerlord

**Pull request closed without merging**